types/bull/index.d.ts

```diff
@@ -414,7 +417,9 @@ declare namespace Bull {
      * If the promise is rejected, the error will be passed as a second argument to the "failed" event.
      * If it is resolved, its value will be the "completed" event's second argument.
      */
-    process(callback: ((job: Job<T>, done: DoneCallback) => void) | ((job: Job<T>) => Promise<any>) | string): void;
+    process(callback: (job: Job<T>, done: DoneCallback) => void): void;
+    process(callback: (job: Job<T>) => Promise<void>): void;
+    process(callback: string): void;
```

Comment:
IMHO - there should be a type definition for the job callbacks.  Since we're pretty repetitive with this and it's a standard pattern in bull.

Reply:
Done 👍🏻